.changie.yaml

```
@@ -0,0 +1,18 @@
+changesDir: .changes
+unreleasedDir: unreleased
+changelogPath: CHANGELOG.md
+versionExt: md
+versionFormat: '## {{.Version}} ({{.Time.Format "January 02, 2006"}})'
+kindFormat: '### {{.Kind}}'
+changeFormat: '* {{.Body}}'
+kinds:
+  - label: BREAKING CHANGES
+  - label: NOTES
+  - label: FEATURES
+  - label: IMPROVEMENTS
+  - label: BUG FIXES
+newlines:
+  afterChangelogHeader: 1
+  beforeChangelogVersion: 1
+  endOfVersion: 1
```

This seems to get the `CHANGELOG.md` format the closest to today (e.g. removing the newline at the beginning of the file, ensuring there are newlines between category titles):

```suggestion
  afterKind: 1
  beforeKind: 1
  endOfVersion: 2
```

Verified via:

```shell
changie new # testing entry
changie batch 3.3.0
changie merge
head CHANGELOG.md
```